src/cpp/src/continuous_batching_pipeline.cpp

```diff
@@ -144,6 +144,8 @@ class ContinuousBatchingPipeline::Impl {
                                                                             m_scheduler->get_config().block_size,
                                                                             m_scheduler->get_config().enable_prefix_caching);
         sequence_group->set_sequence_group_ptr(sequence_group);
+        if (!sampling_params.stop_strings.empty())
+            sequence_group->include_tokenizer(m_tokenizer);
```

Comment:
IMO, adding tokenizer to Sequecegroup look overkill, so can we perform tokenization of `stop_strings` to convert them to `stop_token_ids` and work with `stop_token_ids` only inside `try_finish_generation` ?